package.json

```diff
@@ -120,6 +120,7 @@
     "husky": "8.0.0",
     "jest": "28.1.1",
     "nodemon": "2.0.14",
+    "playwright": "1.22.2",
```

Comment:
2 things there:

- It's better to avoid handling multiple (unrelated) tasks in the same PR. Whenever possible, as PR should only focus on one scope.
- There is no need to add `playwright` dep following the [official documentation](https://playwright.dev/docs/intro#manually). [`playwright`](https://github.com/microsoft/playwright/tree/main/packages/playwright) is just a package linker for `@playwright/test` (synonym of `playwright-test`).